src/components/common/UserProfileCardComponent.js

```diff
@@ -10,7 +10,7 @@ const UserProfileCard = ({
 <Card style={{ width: '38rem', height: '33rem' }} className = 'p-5 user-card'>
 <div className = 'top-content mb-5'>
 <span className = 'float-left'>
-<p className = 'user-name'>{userName}</p>
+<p className = 'user-name'>{userName.replace(/\d+/, '')}</p>
```

Comment:
I believe the logic here is to remove the first set of numbers it finds in the string. Is it not possible for the username to be strictly numbers?